cmds/bmp/sl_command_test.go

```diff
@@ -65,18 +66,183 @@ var _ = Describe("sl command", func() {
 	})
 
 	Describe("#Validate", func() {
-		It("validates a good SlCommand", func() {
-			validate, err := cmd.Validate()
-			Expect(validate).To(BeTrue())
-			Expect(err).ToNot(HaveOccurred())
+		Context("validates a good SlCommand", func() {
+			Context("when --packages specified", func() {
+				BeforeEach(func() {
+					options = cmds.Options{
+						Verbose:  false,
+						Packages: true,
+					}
+				})
+
+				It("validation pass without errors", func() {
+					cmd = bmp.NewSlCommand(options, fakeBmpClient)
+					validate, err := cmd.Validate()
+					Expect(validate).To(BeTrue())
+					Expect(err).ToNot(HaveOccurred())
+				})
+			})
+
+			Context("when --package-options specified", func() {
+				BeforeEach(func() {
+					options = cmds.Options{
+						Verbose:        false,
+						PackageOptions: "1",
+					}
+				})
+
+				It("validation pass without errors", func() {
```

Comment:
This needs in correct English, so, this is better:

'''
It("passes validation without errors", ...)
'''